src/main.cc

```diff
@@ -11,14 +11,48 @@
 
 #ifdef _MSC_VER
 #include <fcntl.h> // For _O_BINARY.
+
+// By default, Windows only looks in the System32 directory for dlls. Npcap installs
+// into System32\Npcap, so we have to add that path to the search path for DLLs or
+// the process won't find it. This is annoying, but it's how the Npcap project
+// recommends we do it. See https://npcap.com/guide/npcap-devguide.html#npcap-feature-native
+// for further info.
+static void init_npcap_dll_path()
```

Comment:
Only thought was whether this could live closer to iosource/pcap and called from an early plugin hook, but seems BPF_Program and maybe others would need it, too. And may be too early for a win.cc file.